packages/expo-screen-orientation/android/src/main/java/expo/modules/screenorientation/ScreenOrientationModule.java

```diff
@@ -0,0 +1,282 @@
+package expo.modules.screenorientation;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.ActivityInfo;
+import android.util.DisplayMetrics;
+import android.view.Surface;
+import android.view.WindowManager;
+
+import org.unimodules.core.ExportedModule;
+import org.unimodules.core.ModuleRegistry;
+import org.unimodules.core.Promise;
+import org.unimodules.core.errors.InvalidArgumentException;
+import org.unimodules.core.interfaces.ActivityProvider;
+import org.unimodules.core.interfaces.ExpoMethod;
+import org.unimodules.core.interfaces.LifecycleEventListener;
+import org.unimodules.core.interfaces.services.UIManager;
+
+public class ScreenOrientationModule extends ExportedModule implements LifecycleEventListener {
+  private ActivityProvider mActivityProvider;
+  private Integer mInitialOrientation = null;
+  static final String ERR_SCREEN_ORIENTATION = "ERR_SCREEN_ORIENTATION";
```

Comment:
If you would be a user of Expo, would you be able to tell me what this error code means?